ui/pages/asset/components/chart-tooltip.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { forwardRef, useImperativeHandle, useState } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  Box,
+  Text,
+  TextDirection,
+} from '../../../components/component-library';
+import { getCurrentCurrency } from '../../../selectors';
+import { formatCurrency } from '../../../helpers/utils/confirm-tx.util';
+import {
+  TextAlign,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import { getPricePrecision } from './util';
+
+// A label indicating the minimum or maximum price on the chart
+const ChartTooltip = forwardRef((_, ref) => {
```

Comment:
why do we have unused first param here ?

Reply:
It was because the component didn't accept any props, but now it does.